A59-lrs-named-metrics.md

```diff
@@ -0,0 +1,97 @@
+A59: xDS LRS Named Metrics Support
+----
+* Author: yousukseung
+* Approver(s): markdroth
+* Status: Ready for Implementation
+* Implemented in: C-core
+* Last updated: 2023-05-10
+* Discussion at: TBA
+
+## Abstract
+
+This proposal describes how the gRPC xDS client will support named metrics from the backends in load reports to the LRS server at the locality level.
+
+## Background
+
+The gRPC xDS client currently includes request counts in LRS load reports. These are locally counted simple request counts, however there are cases where the service needs to include custom data from the backends in LRS load reports for load balancing purposes. gRPC now supports ORCA based load reporting from the backend and this proposal describes how to integrate it with LRS load reporting in the gRPC xDS client.
```

Comment:
There are some grammar issues in this paragraph.  Suggest writing this as follows:

"""
The initial xDS functionality in the gRPC client described in [gRFC A27][A27] includes the ability to report load to the control plane via LRS.  However, those load reports currently contain only basic request counts tracked by the client.  [gRFC A51][A51] added support for servers to report custom backend metrics to the client via the ORCA protocol.  This proposal adds the ability for the client to propagate those backend metrics to the control plane in the LRS load reports.
"""

Reply:
Updated. For consistency, I updated elsewhere to use "custom metrics" instead of "named metrics", but updated "servers" with "backends" from your edit.